docs/getting-started.rst

```diff
@@ -0,0 +1,74 @@
+===============
+Getting Started
+===============
+
+Installing Prerequisite Software
+--------------------------------
+
+SALib requires `NumPy <http://www.numpy.org/>`_, `SciPy <http://www.scipy.org/>`_,
+and `matplotlib <http://matplotlib.org/>`_ installed on your computer.  Using
+`pip <https://pip.pypa.io/en/stable/installing/>`_, these libraries can be
+installed with:
+
+::
+
```

Comment:
These three steps are not necessary, as the command `pip install SALib` automatically installs the dependencies too!